internal/buildscripts/update-deps

```diff
@@ -34,7 +34,7 @@ for gomod in $( find "$REPO_DIR" -name "go.mod" | grep -v "/examples/" | sort );
     fi
 
     set +o pipefail
-    lines="$( grep 'github.com/open-telemetry/opentelemetry-collector-contrib/' "$gomod" | grep -v '=>' | grep -v ' // indirect' | sort -u )"
+    lines="$( grep 'github.com/open-telemetry/opentelemetry-collector-contrib/' "$gomod" | grep -v '^[[:space:]]*//' | grep -v '=>' | grep -v ' // indirect' | sort -u )"
```

Comment:
Are these couple changes related to the PR?

Reply:
I use this script for the dep updates and needed to ignore the comments.